Assets/Scripts/UI/ButtonManager.cs

```diff
@@ -61,8 +61,7 @@ public void OnStatsButton()
 
     public void OnResetButton()
     {
-        PlayerPrefs.DeleteAll();
-        File.Delete(SaveSystem.path);
+        SaveSystem.DeleteAll();
         SceneManager.LoadScene("Scenes/MainLevel");
```

Comment:
"Scenes/MainLevel" put the value in a separate field, do not hardcode

Reply:
Fixed